echo/server.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
```

Comment:
nit: can we use the V4?

Reply:
+1 @afik 

to give a better context, there is a breaking changes between v1 and v4. So far, AFAIK if you use v1 component, you couldn't use v4 component and vice versa :sob:

Reply:
Hoo, thankss. That actually was added by the IDE, didn't realized it's different version hehe